src/node/desktop/src/ui/widgets/choose-r/preload.ts

```diff
@@ -132,3 +166,7 @@ const callbacks: Callbacks = {
 };
 
 contextBridge.exposeInMainWorld('callbacks', callbacks);
+
+function isRVersionSelected(selectedVersion: string, versionToCompare: string) {
+  return threatPathString(selectedVersion) === threatPathString(versionToCompare);
```

Comment:
If I understand correctly, you want to compare the paths after normalization. If that's the case, I think using `normalizeSeparators()` is appropriate here.

Reply:
I just want to make sure both are equal, with no `\`